src/components/starlight/EditLink.astro

```diff
@@ -12,10 +12,15 @@ const githubEditUrl =
 
 {
 	editUrl && (
-		<a href={githubEditUrl} class="sl-flex">
-			<Icon name="pencil" size="1.2em" />
-			{labels['page.editLink']}
-		</a>
+		<div class="sl-flex">
+			<a href={githubEditUrl} class="sl-flex" style="padding-right: 10px;">
+				<Icon name="pencil" size="1.2em" />
+				{labels['page.editLink']}
+			</a>
+			<a href="https://contribute.docs.astro.build/guides/i18n/" class="sl-flex" style="padding-left: 10px;">
+				<Icon name="translate" size="1.2em" /> Translate
```

Comment:
Question: Should this not be translated?

Reply:
Same as before. I can review this. Perhaps I missed the translation for this.